### PR TITLE
CBAPI-3371: Bump version number to 1.3.5

### PR DESCRIPTION
**Pull request by abowersox-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-3371

## Pull Request Description

Update release number to 1.3.5 and create Changelog entries for new release, in accordance with standard release procedure.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

Doc updates only, no code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # VMware Carbon Black Cloud Python SDK
 
-**Latest Version:** 1.3.4
+**Latest Version:** 1.3.5
 <br>
-**Release Date:** October 12, 2021
+**Release Date:** TBD
```
Comment (avanbrunt-cb) [inline README.md:5]: Can we pick a date? Or is this PR going to be held until security tools finish?

Comment (kebringer-cb) [reply README.md:5]: Coverity & Blackduck have been reviewed this morning, both are ok.

Comment (abowersox-cb) [reply README.md:5]: According to our SDK release process, the date is filled in on the new release branch once release testing is completed, and then merged back to the `develop` branch.

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
 ================================
+CBC SDK 1.3.5 - Released (TBD)
+------------------------------
+
+New Features:
+
+* Added asynchronous query support to Live Query.
+* Added the ability to export query results from Live Query, either synchronously or asynchronously (via the ``Job``
+  object and the Jobs API).  Synchronous exports include full-file export, line-by-line export, and ZIP file export.
+  Asynchronous exports ionclude full-file export and line-by-line export.
```
Comment (kebringer-cb) [inline docs/changelog.rst:11]: Typo:  ionclude

File: README.md (hunk 1 above)
Comment (avanbrunt-cb) [reply README.md:5]: Thanks for the clarification

**Review by avanbrunt-cb — APPROVED** [review]

File: docs/changelog.rst (hunk 2 above)
Comment (abowersox-cb) [reply docs/changelog.rst]: fixed

**Review by lhilmes-cb — APPROVED** [review]

**Pull request merged**